versioned_docs/version-2.0.x/reference/goals/awslambda.mdx

```diff
@@ -53,19 +53,5 @@ None
 
 None
 
-## Related subsystems
-
-- [download-pex-bin](../subsystems/download-pex-bin.mdx)
-- [grpc_python_plugin](../subsystems/grpc_python_plugin.mdx)
-- [lambdex](../subsystems/lambdex.mdx)
-- [pex](../subsystems/pex.mdx)
-- [protoc](../subsystems/protoc.mdx)
-- [python-infer](../subsystems/python-infer.mdx)
-- [python-native-code](../subsystems/python-native-code.mdx)
-- [python-protobuf](../subsystems/python-protobuf.mdx)
-- [python-repos](../subsystems/python-repos.mdx)
-- [python-setup](../subsystems/python-setup.mdx)
-- [source](../subsystems/source.mdx)
-- [subprocess-environment](../subsystems/subprocess-environment.mdx)
```

Comment:
This is lost?

Reply:
I think so

Reply:
Fixed in https://github.com/pantsbuild/pantsbuild.org/pull/53